private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -1174,6 +1262,32 @@ export class TemplateServiceClient {
       let tags: string[] | string = req.body.tags;
       let data: JSON[] | JSON = req.body.data;
 
+      let bindData: boolean | undefined = req.body.bindData ? req.body.bindData.toLowerCase() === "true" : undefined;
```

Comment:
Are we adding this functionality to the client sdk? If yes do you mind also adding it to the swagger definition file?

Reply:
As discussed, let's leave it out for right now. I don't see the need.